internal/controller/policymanifest_controller.go

```diff
@@ -83,10 +82,11 @@ func (r *PolicyManifestReconciler) Reconcile(ctx context.Context, req ctrl.Reque
 	policyMap := make(map[string]kyvernov1.ClusterPolicy)
 
 	var kyvernoPolicy kyvernov1.ClusterPolicy
-	if err := r.Get(ctx, types.NamespacedName{Namespace: "", Name: polman.Name}, &kyvernoPolicy); err != nil {
-		// Error fetching the report
-		log.Log.Error(err, "unable to fetch Kyverno Policy")
-		return ctrl.Result{}, client.IgnoreNotFound(err)
+	var ok bool
+
+	if kyvernoPolicy, ok = r.PolicyCache[polman.Name]; !ok {
```

Comment:
You don't need to declare the variables if you assign them in the `if` condition.

```
	if kyvernoPolicy, ok := r.PolicyCache[polman.Name]; !ok {
```